recipes/mikktspace/all/CMakeLists.txt

```
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.4)
+project(mikktspace)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(MSVC AND BUILD_SHARED_LIBS)
```

Let's enable it for mingw too.
Btw, the conditional is not necessary, because it can only export symbols for shared libraries.
```suggestion
if(WIN32 AND BUILD_SHARED_LIBS)
```

---

I know the conditional is useless, but I think it's better when it's explicit.